doc/man3/OSSL_PARAM_TYPE.pod

```diff
@@ -292,6 +410,13 @@ could fill in the parameters like this:
     OSSL_PARAM_set_utf8_ptr(OSSL_PARAM_locate(params, "foo"), "foo value");
     OSSL_PARAM_set_utf8_string(OSSL_PARAM_locate(params, "bar"), "bar value");
 
+or, more succinctly, like this:
+
+    /* const OSSL_PARAM *params */
+
+    OSSL_PARAM_locate_set_utf8_ptr(params, "foo", "foo value");
+    OSSL_PARAM_locate_set_utf8_string(params, "bar", "bar value");
+
```

Comment:
If you rebase, you will notice that the optional calls are more suitable

Reply:
A mixture of both perhaps?

Reply:
I've done a mixture.  The example had the wrong check against NULL :(